grails-app/conf/Config.groovy

```diff
@@ -93,6 +93,10 @@ googleAnalytics.trackingId = null
 // set per-environment serverURL stem for creating absolute links
 def env = System.getenv()
 
+filtering {
+    loadFromFiles = Boolean.parseBoolean(env['LOAD_FILTERS_FROM_FILES'])
```

Comment:
Happy to take suggestions for the env var name. Although, I am happy enough with this.